cmd/down.go

```diff
@@ -27,11 +27,6 @@ func down(cmd *cobra.Command, args []string) {
 	//read the harbor compose file
 	var harborCompose = DeserializeHarborCompose(HarborComposeFile)
 
-	_, _, err := Login()
```

Comment:
I say just leave this as is. HarborApi relies on auth so i can see it go either way.